demos/python_demos/multi_camera_multi_person_tracking/run_evaluate.py

```diff
@@ -0,0 +1,151 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import argparse
+import json
+import logging as log
+
+import motmetrics as mm
+import numpy as np
+from lxml import etree
```

Comment:
Is the built-in `xml.etree.ElementTree` module insufficient here?

Reply:
Yes, it is enough. I did not now about built-in xml parser. `lxml` removed.